raco/myrial/exceptions.py

```diff
@@ -162,3 +162,11 @@ def __init__(self, relname):
 
     def __str__(self):
         return "No such relation: %s" % self.relname
+
+
+class InvalidSamplingTypeException(MyrialCompileException):
```

Comment:
Not sure this merits a new expression, why not just `raise` a `MyrialCompileException` with that error message